autoclose.rules.mjs

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
I would put this file rules file and the script inside a new folder at the path *.github/actions/autoclose-issues/* because is only part of our CI. Adding it to the root and the script to the root of the *scripts/* folder seems to me a bit confusing.

Reply:
Or probably put the rules file into *.github/* folder.

Reply:
I would just remove `.github export-ignore` from *.gitattributes*.

Reply:
> dot-files (except for .eslintrc.*) as well as dot-folders and their contents are ignored.
> https://eslint.org/docs/latest/use/configure/ignore-deprecated

It looks like we can't unignore the `.github` folder. I also tried some methods in #13015, but unfortunately, none worked.

Reply:
🤔 In that case, I would put both files in a new folder at *scripts/autoclose-issues/*. I'll try to debug the problem in the future. Reading the documentation of ESLint v9, it seems to me that it could be possible or should reported as a bug.

Reply:
Ok, I've moved those files to `scripts/autoclose-issues/`.